skeleton-typescript/build/bundles.js

```diff
@@ -29,8 +29,7 @@ module.exports = {
         "aurelia-logging-console",
         "bootstrap",
         "bootstrap/css/bootstrap.css!text",
-        "fetch",
-        "jquery"
```

Comment:
tried to bundle? usually bootstrap complaints or the dropdown won't work. i'd suggest you rather add jquery to the package.json or something, rather then to remove it here